website/content/en/docs/building-operators/golang/advanced-topics.md

```diff
@@ -285,6 +285,60 @@ func main() {
 
 When the operator is not running in a cluster, the Manager will return an error on starting since it can't detect the operator's namespace in order to create the configmap for leader election. You can override this namespace by setting the Manager's `LeaderElectionNamespace` option.
 
+### Support for multiple architectures
```

Comment:
question, does SDK support multi-arch operator builds currently in any way?

Reply:
Not really. There is nothing in the SDK preventing the release of operators running on multiple architectures but to the best of my knowledge there is nothing supporting it either. With "Support" I meant here the operator not the SDK. I could rename the chapter to "Multiple architectures" or "Operator support for multiple architecture" as it seems confusing.

Reply:
Thinking of it. I could also reference the docker way of doing multi-arch builds. If we wanted the SDK to support multi-arch builds of the operator the Makefile would need to be amended for it. Also the generated Dockerfile for go operators is explicitly referencing GOARCH=amd64. With docker TARGETPLATFORM could be used, which is  automatically set to what is provided by the --platform flag of buildx. This does not come out of the box with buildah yet.

Reply:
Agreed that Makefile and dockerfile could have amends such as setting `--platform` flag to `TARGETPLATFORM`. 

And modify the `Makefile` to build the docker image, the command will be: 
`docker buildx build --platform linux/amd64,linux/arm64 -t ${IMG}`.

Also, `--platform` is a docker buildx feature, I am not sure that it work with `docker build`.

Reply:
Yes multi-arch builds is supported by docker buildx not docker build. I would propose the following:
1. with this PR amending the documentation for the state as is
2. Creating an issue where we can discuss operator-sdk enhancements. I think that operator-sdk init could be extended to generate Makefile and Dockerfile with multi-arch capabilities based on a flag. Another similar thing would be to have the Makefile using podman/buildah rather than docker based on another flag. It seems that the option was previously available with --image-builder but disappeared when the build moved to Makefile (the user needs now to update the Makefile manually)

Reply:
I created this feature request: https://github.com/operator-framework/operator-sdk/issues/5180